Gemfile

```diff
@@ -4,6 +4,7 @@ gem 'active_model_serializers', '~> 0.9.3'
 gem 'active_skin'
 gem 'activeadmin', '~> 1.3.0'
 gem 'activeadmin_addons'
+gem 'aasm'
```

Comment:
Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem aasm should appear before activeadmin_addons.